lua/trouble/init.lua

```diff
@@ -53,9 +53,6 @@ end
 
 function Trouble.open(...)
   local opts = get_opts(...)
-  if opts.mode and (opts.mode ~= config.options.mode) then
-    config.options.mode = opts.mode
```

Comment:
As far as I can see, this makes setting a mode in the config pointless as it will be overwritten anyway?